<launch>
    <remap from="/global_nozzle_pose" to="/mur620/UR10_l/global_tcp_pose" />
    <remap from="/ur_cmd" to="/mur620/UR10_l/twist_fb_command" />
    <!-- <node name="pose_relay" pkg="topic_tools" type="relay" args="/mur620/UR10_l/global_tcp_pose /global_nozzle_pose" /> -->
    <include file="$(find ur_trajectory_follower)/launch/combine_twists.launch" />
    <node name="twist_combiner_world" pkg="ur_trajectory_follower" type="combine_twists.py" output="screen">
        <rosparam param="twist_topics" subst_value="true">['/orthogonal_twist', '/ur_twist_direction_world']</rosparam>
        <remap from="combined_twist" to="/ur_twist_world"/>
    </node>
    <include file="$(find ur_trajectory_follower)/launch/compensate_mir.launch" />
    <include file="$(find ur_trajectory_follower)/launch/path_idx_advancer.launch" />
    <include file="$(find ur_trajectory_follower)/launch/transform_twist_to_mir.launch" />
    <include file="$(find ur_trajectory_follower)/launch/ur_direction_controller.launch">
        <arg name="ff_only" value="true" />
        <remap from="/ur_twist_world" to="/ur_twist_direction_world" />
    </include>
    <include file="$(find ur_trajectory_follower)/launch/orthogonal_error_correction.launch"/>

</launch>